python/python/glide/async_commands/core.py

```diff
@@ -3266,6 +3266,111 @@ async def xautoclaim_just_id(
             await self._execute_command(RequestType.XAutoClaim, args),
         )
 
+    async def xinfo_groups(
+        self,
+        key: str,
+    ) -> List[Mapping[str, Union[str, int, None]]]:
+        """
+        Returns the list of all consumer groups and their attributes for the stream stored at `key`.
+
+        See https://valkey.io/commands/xinfo-groups for more details.
+
+        Args:
+            key (str): The key of the stream.
+
+        Returns:
+            List[Mapping[str, Union[str, int, None]]]: A list of mappings, where each mapping represents the attributes
+                of a consumer group for the stream at `key`. Each mapping contains the following info:
+                - name: the consumer group's name.
```

Comment:
Do we need to list these? `xinfo stream` has way more fields returned

Reply:
Yeah I was wondering whether I should include them or not, I'm fine either way

Reply:
I would recommend removing these and letting users click on the handy links provided. 
If Valkey changes the return, our doc will be out of date.